golang/go-cloud-run-hello-world/index.html

```diff
@@ -0,0 +1,207 @@
+<!doctype html>
```

Comment:
This file was copied as-is from cloud-run-hello. There are open issues in the repo to tune it for accessibility so we'll want to track and port changes

Reply:
Also, this page has gotten complicated as it emphasizes a strong out-of-box experience as a running app, rather than being a sample HTML page. Do we want to maintain this as-is in multiple languages or simplify?

Reply:
When you say simplify - do you mean having the same template HTML referenced in multiple languages templates?

Reply:
The current webpage involves multiple images, complicated CSS, support for dark mode, etc. It is complicated enough that it's had a number of changes recently to improve accessibility, performance, and so on: in other words, it's not really maintained as a sample but as an experience.

Do we want to track the cloud-run-hello or fork? If we want to fork, we should consider simplifying. For example, instead of the full web page currently generated, we could go with:

text/plain:
```
Service ??? running Revision ??? in Project ???
```

I asked this question because I don't have a strong sense of the value of the more visual experience in this case.

Reply:
The value of more visual experience is giving developers better sense of accomplishment. It also demoes better compared to a plain text.
I agree, we should fork and simplify where appropriate, we don't need a dark mode or other more advanced UI-related scenarios or features.